vocab.py

```diff
@@ -20,9 +28,102 @@ class ValidDataSourceName(Enum):
     Each data source name should be defined in ValidDataSourceName.
     NAME_OF_DATA_SOURCE = auto()
     """
-    
+
     CANVAS_API = auto()
     CANVAS_ZOOM_MEETINGS = auto()
     KALTURA_API = auto()
     UNIZIN_DATA_PLATFORM_EVENTS = auto()
     UNIZIN_DATA_WAREHOUSE = auto()
+
+    def __str__(self) -> str:
+        '''
+        Return value name without `ValidDataSourceName` class name.
+
+        :return: Name of enum value.
+        '''
+        return str(self.name)
+
+
+class DataSourceStatus:
+    '''
+    Each data source process must return an instance of this class, which tells the name of the
+    data source and the time at which data from it was last updated.
+    '''
+
+    def __init__(
+            self,
+            data_source_name: ValidDataSourceName = NotImplemented,
+            data_updated_at: datetime = NotImplemented
```

Comment:
I've changed this section of code to not use Pandas.

As you point out, `pd.Timestamp` objects can often be used like `datetime` ones, thanks to Python duck-typing.  So that other type can be passed as an argument and it will work.